src/Microsoft.AspNet.Mvc.Razor/RazorPage.cs

```diff
@@ -68,6 +68,12 @@ public HttpContext Context
         [Activate]
         public IHtmlEncoder HtmlEncoder { get; set; }
 
+        /// <summary>
+        /// Gets the <see cref="IJsonHelper"/> to be used for serializing JSON.
+        /// </summary>
+        [Activate]
+        public IJsonHelper Json { get; set; }
```

Comment:
We support whatever base class the Razor author chooses for their view; `RazorPage` is just the default.  Since the `RazorPage` class does not need this property, can be injected.  That is, revert this and instead change `_defaultInheritedChunks` in [`MvcRazorHost`](https://github.com/aspnet/Mvc/blob/dev/src/Microsoft.AspNet.Mvc.Razor.Host/MvcRazorHost.cs#L33).